contracts/handlers/ERC20Handler.sol

```diff
@@ -22,6 +27,18 @@ contract ERC20Handler is IDepositExecute, HandlerHelpers, ERC20Safe {
     // depositNonce => Deposit Record
     mapping (uint8 => mapping(uint256 => DepositRecord)) public _depositRecords;
 
+    /**
+        @param bridgeAddress Contract address of previously deployed Bridge.
+        @param initialResourceIDs Resource IDs are composed of chain ID + contract address, and used to identify a specific contract address.
```

Comment:
Please remove composition of resource ID, not necessarily the case if not originating from eth

Reply:
```suggestion
        @param initialResourceIDs Resource IDs are used to identify a specific contract address.
```